src/Seeds/Data/detections.json

```diff
@@ -0,0 +1,11 @@
+[{
+  "class": "boat",
+  "score": 1,
+  "timestamp": "5/1/2021 8:30:52AM"
+},
+{
+"class": "glass",
+"score": 7,
+"timestamp": "5/1/2021 8:30:52AM"
+}]
+
```

Comment:
Should we consider adding more items here? Maybe about 50 of them?

Reply:
done